config/hero.yaml

```
@@ -21,10 +21,10 @@ shortcuts:
     button:
       href: /contributing
       text: Contribute to the project
-  - title: Want to use AEPs for your organization?
+  - title: Want to use AEPs in your organization?
```

```suggestion
  - title: Want to adopt AEPs in your organization?
```

nit: since it's an API design specification, maybe "adopt" is a better verb?